src/store/getters.ts

```diff
@@ -1081,6 +1082,18 @@ export default {
     return state.stateModel.stateFilingRestoration
   },
 
+  getRelationships (state: StateIF): RelationshipTypes[] {
```

Comment:
I think this getter is too granular, since relationships only apply to restoration filings.

AFAIK, a computed method (local getter) is also cached and reactive, so having the getter here doesn't buy us anything.

Thoughts?

Reply:
If we're going to follow best practices and not repeat ourselves then any method that is repeated in more than one place needs to be moved to location that's common.  In this particular case `getRelationships()` is required in both the YourCompanySummary and the LimitedRestorationtoFull so it makes sense to move it to the store.  We could, if you prefer, move it somewhere else that's common (like a mixin).  In general, I think it business logic should not be located in components.  It makes it difficult to reuse and hence more likely to be copied and duplicated elsewhere.

Reply:
My first thought was that I'd like to reduce our use of mixins and save them for non-store data.

My second thought is that it might be nice to have "something" specific to restorations. Surely not a store module. Maybe a mixin? I dunno.

Given that this getter is used in a couple of places, I'm OK with the current implementation.